en/debug/probe_stlink.md

```diff
@@ -0,0 +1,197 @@
+
+Carefully remove the two GPIO1/2 cables from one connector of the cable.
+Then remove all cables from the other connector.
+You are left with a eight crimped connectors at the end of the wires.
+
+![](../../assets/debug/stlinkv3_minie_p7.jpeg)
+
+Tin the crimped cable connectors and BTB connector and solder the crimped connectors directly to the STLinkv3.
+Be careful to not create shorts between the cables, as the crimped connectors are quite large.
+
+![](../../assets/debug/stlinkv3_minie_p8.jpeg)
+
+### Testing
+
+You should now test your debug probe to ensure you do not have any electrical shorts.
+
+1. Plug the probe into your target via the Pixhawk Debug Port.
+2. Test the serial port with a program of your choice.
+3. Test the SWD and RST connection via [OpenOCD][] or [STLink][] software.
+4. Test the SWO connection via [Orbuculum][].
+
+See the [Embedded Debug Tools][emdbg] for more information about software support for the PX4 FMUv5 and FMUv6 flight controllers.
+
+### Make it Smaller
+
+This step removes the 14-pin debug interface on the back of the STLinkv3-MINIE and adds shrink tubing around the entire device to improve handling and prevent shorting the STLink against metal parts or PCBs.
+This step is strictly optional and requires:
+
+- 1x 20mm shrink tubing about 5cm long.
+- 1x flat tongs to hold the STLinkv3 by the USB-C port.
+- 1x fine cutting pliers or soldering iron.
+- 1x heat gun.
+
+Use the pliers to gently pull off the plastic part of the STDC14 connector.
+This leaves you with only the connector pins.
+
+![](../../assets/debug/stlinkv3_minie_p9.jpeg)
+
+Using the fine pliers, cut off the connector pins being very careful not to damage the PCB or any components on the PCB.
+Alternatively, you can solder these connector pin off the PCB, but it can take longer.
+
+![](../../assets/debug/stlinkv3_minie_p10.jpeg)
+
+Rotate the STLinkv3 to cut off the other row, again being very careful to not damage it.
+
+![](../../assets/debug/stlinkv3_minie_p11.jpeg)
+
+Cut a ~5cm (~2in) long piece of shrink tube.
+It should be flush with the USB-C connector and extend a little beyond the end.
+
+![](../../assets/debug/stlinkv3_minie_p12.jpeg)
+
+Hold both the PCB and the shrink tube with the flat tongs by the **bottom** metal part of the USB-C connector.
+Be careful not to accidentally squeeze the middle plastic part of the USB-C connector!
+
+![](../../assets/debug/stlinkv3_minie_p13.jpeg)
+
+Use the heat gun to shrink the tubing all around the debug probe.
+Make sure the tubing is equally shrunk and protects the whole PCB.
+
+![](../../assets/debug/stlinkv3_minie_p14.jpeg)
+
+Optionally, you may add a logo of your choice printed on paper and cut to size.
+Be aware that the heat can make the ink flow a little, so you may need to experiment with what settings work with your printer.
+
+![](../../assets/debug/stlinkv3_minie_p15.jpeg)
+
+[stlinkv3]: https://www.st.com/en/development-tools/stlink-v3minie.html
```

Comment:
FYI this kind of linking is not used very often in the user guide. Generally we link once externally and then don't subsequently - this is reserved mostly for use in images in tables, where space is a premium. Not a big deal - the tools render it OK.